src/SFA.DAS.EmployerFinance.AcceptanceTests/Steps/AccountSteps.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Threading.Tasks;
+using BoDi;
+using HMRC.ESFA.Levy.Api.Client;
+using HMRC.ESFA.Levy.Api.Types;
+using Moq;
+using SFA.DAS.EmployerFinance.AcceptanceTests.Extensions;
+using SFA.DAS.EmployerFinance.AcceptanceTests.TestRepositories;
+using SFA.DAS.EmployerFinance.Data;
+using SFA.DAS.EmployerFinance.Models.Paye;
+using TechTalk.SpecFlow;
+
+namespace SFA.DAS.EmployerFinance.AcceptanceTests.Steps
+{
+    [Binding]
+    public class AccountSteps : TechTalk.SpecFlow.Steps
+    {
+        private readonly IObjectContainer _objectContainer;
+        private readonly ObjectContext _objectContext;
+
+        public AccountSteps(IObjectContainer objectContainer, ObjectContext objectContext)
+        {
+            _objectContainer = objectContainer;
+            _objectContext = objectContext;
+        }
+
+        [Given(@"We have an account")]
+        public Task GivenWeHaveAnAccount()
+        {
+            return _objectContainer.ScopeAsync(c => _objectContext.CreateAccount(c));
+        }
+
+        [Given(@"We have an account with a paye scheme")]
+        public Task GivenWeHaveAnAccountWithPayeScheme()
+        {
+            return _objectContainer.ScopeAsync(async c =>
+            {
+                await _objectContext.CreateAccount(c);
+                await InitialisePayeSchemeRef(c, "123/ACT");
+            });
+            //return _objectContainer.ScopeAsync(c => 
+            //           _objectContext.CreateAccount(c)
+            //                    .ContinueWith(t => InitialisePayeSchemeRef(c, "123/ACT")));
```

Comment:
Does this commented out code need to be removed?

Reply:
Done